README_zh-CN.md

```diff
@@ -56,6 +56,10 @@ MMAction2 是一款基于 PyTorch 的视频理解开源工具箱，是 [OpenMMLa
   <img src="https://github.com/open-mmlab/mmaction2/raw/master/resources/spatio-temporal-det.gif" width="800px"/><br>
     <p style="font-size:1.5vw;">AVA-2.1 上的时空动作检测</p>
 </div>
+<div align="center">
+  <img src="https://user-images.githubusercontent.com/30782254/155670506-13fd3990-ce2b-4b55-a7e3-f24277d2f8d8.gif" width="800px"/><br>
+    <p style="font-size:1.5vw;">AVA-2.1 上的基于 skeleton 的时空动作检测和动作识别</p>
```

Comment:
There are several problems:
1. Is this video really from AVA-2.1? I don't think so.
2. There are some noisy predictions beyond the two main subjects, can we find a better demo video? 
3. The gif size is too large, can we put it and the original skeleton demo in the same line? You can write html to do so, or merge the two gifs.